apps/prairielearn/python/prairielearn/html_utils.py

```diff
@@ -17,6 +17,22 @@
 
 EnumT = TypeVar("EnumT", bound=Enum)
 
+LIBXML_BOOLEAN_ATTRIBUTES = frozenset({
```

Comment:
Want to link to the source of this list? https://gitlab.gnome.org/GNOME/libxml2/-/blob/4aa08c80b711ab296f6e6ecab24df8cf6d0be5fc/HTMLtree.c#L306

Reply:
Done in d16b021c8ca983635056641cff3c80539a453d39.